### PR TITLE
feat: view transactions by program id

### PR DESCRIPTION
**Pull request by sunnygleason**
## Problem

* Transactions overall are too high-volume to watch
* We need a way to view transactions by program id (for example, tic-tac-toe)
* Todo: implement #12

## Solution

* Introduce a txn-by-program-id topic per program id
* Make it easy to filter by account id on client- or server-side

### CODE REVIEW

File: api/inbound-stream.js [hunk 1]
```diff
@@ -226,6 +249,9 @@ class RedisHandler {
         // increment txn hour count
         commands.push(['incrby', `!txn-per-hour:${txn_hour}`, txCount]);
 
+        // increment txn day count
+        commands.push(['incrby', `!txn-per-day:${txn_day}`, txCount]);
```
Comment (sunnygleason) [inline api/inbound-stream.js]: added a "daily" aggregation to support that zoom level when we get there

File: api/api.js [hunk 2]
```diff
@@ -240,4 +297,43 @@ app.get('/txn/:id', (req, res) => {
   });
 });
 
+app.get('/search/:id', (req, res) => {
+  client.exists(`!txn:${req.params.id}`, (e1, v1) => {
```
Comment (mvines) [inline api/api.js]: callback hell!   You don't wanna use `await` here?

**Comment by CriesofCarrots** [comment]
re: eb0aef7 , `git pull --rebase upstream master` ?
Generally we prefer rebasing to merge commits (and it would have caught that comma ;) )

**Pull request merged**